### PR TITLE
Feature/zoom

### PR DESCRIPTION
**Pull request by RV-Adrian**

### CODE REVIEW

File: src/sections/presentations.scss [hunk 1]
```diff
@@ -212,6 +226,7 @@ $panel-width: 290px;
     -webkit-transition: all 0.5s cubic-bezier(0.175, 0.885, 0.32, 1.275);
     /* Safari 3.1 to 6.0 */
     transition: all 0.5s cubic-bezier(0.175, 0.885, 0.32, 1.275);
+    perspective: 800px;
```
Comment (RV-Adrian) [inline src/sections/presentations.scss]: This is a CSS hack to stop glitches appearing when Placeholders are hovered over.

File: src/sections/presentations.scss [hunk 2]
```diff
@@ -554,7 +569,11 @@ $panel-width: 290px;
   -moz-box-shadow: 0 0 0 10000px #d4d4d4;
   box-shadow: 0 0 0 10000px #d4d4d4;
   border: 12px solid #d4d4d4;
-  border-top-width: 60px;
+  border-top-width: 0px;
```
Comment (RV-Adrian) [inline src/sections/presentations.scss]: New spacing to make the Zoom levels not move off-screen.

**Comment by RV-Adrian** [comment]
@ghislaineguerin Please review CSS added to support Zoom for Editor: https://apps-stage-1.risevision.com

There's a couple of inline notes for clarification. Thanks!

**Comment by ghislaineguerin** [comment]
👍 looks good

**Pull request merged**